src/OpenTelemetry/Metrics/PeriodicExportingMetricReader.cs

```diff
@@ -111,14 +111,14 @@ protected override void Dispose(bool disposing)
 
         private void ExporterProc()
         {
-            var sw = Stopwatch.StartNew();
+            int index;
+            int timeout;
```

Comment:
Is there benefit of extracting the `time` out? (e.g. having scoped local variables could help improving the readability).

Reply:
Ignoring this for this PR... just clicked merge. We can address this in followup, if needed.